templates/thank-you.html

```diff
@@ -32,4 +32,24 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
```

Comment:
I would probably line these up to avoid any confusion that there is view-answers and a view-submitted-response feature so view-submitted-response-title rather than view-answers-title. And I would go view-submitted-response-warning etc as well

Reply:
Have aligned these 👍